public/static/locales/en/tenants.json

```diff
@@ -0,0 +1,31 @@
+{
+    "nitrosb":{
+        "title":"Plant Trees with Nitro Snowboards",
+        "description":"Nitro Snowboards set a goal to support and mobilize the conservation, restoration, and growth of a forest."
+    },
+    "energizer":{
+        "title":"Energizer ❤️ Baeume",
+        "description":"`Das Richtige tun` ist für uns der zentrale Leitgedanke, an dem wir unser Handeln ausrichten. Seit der ersten quecksilberfreien Batterie im Jahre 1989 gelingt es uns, unsere Batterien immer weiter zu verbessern, langlebiger und damit umweltfreundlicher zu machen. Unsere Partnerschaft mit Plant-for-the-Planet steht genau in dieser Tradition, die Umwelt zu schonen und für mehr Nachhaltigkeit einzustehen. Gemeinsam pflanzen wir Bäume und veranstalten Bildungsprogramme für Kinder und Jugendliche, um ein Bewusstsein für globale Gerechtigkeit und die Klimakrise zu schaffen. Einbinden wollen wir dabei auch unsere Konsumenten und den Handel, denn nur gemeinsam kann diese zentrale Herausforderung unserer Zeit gelöst werden. Bist du dabei?"
+    },
+    "senatDerWirtschaft":{
+        "title":"Senat Der Wirtschaft",
+        "description":"Die Klimakrise ist eine der größten gesellschaftlichen Herausforderungen und es muss sich etwas ändern. Der Senat der Wirtschaft ist eine Wertegemeinschaft. Wir stehen für eine motivierende und verbindende Vision, verknüpft mit praktischer Umsetzung. Deshalb pflanzen immer mehr unserer Mitglieder Wälder, zusammen mit ihren Mitarbeiter, Geschäftspartnern und Kunden. Jeder gepflanzte Baum bindet CO2 und schenkt uns Menschen wertvolle Zeit. Diese Zeit werden wir nutzen, um unsere CO2-Emissionen massiv zu reduzieren. Versprochen!"
+    },
+    "pampers":{
+        "title":"Pampers ❤️ Wald",
+        "descriptionTitle":"Bäume pflanzen für eine nachhaltigere Zukunft",
+        "description":"Gemeinsam mit euch wollen wir dabei helfen, unseren Kleinsten eine bessere Zukunft zu ermöglichen. Deshalb arbeiten wir mit der Kinder- und Jugendorganisation Plant-for-the-Planet an einem tollen Projekt: dem Pampers Wald. Pro verkaufter Packung Pampers Windeln spenden wir 0,01 EUR/CHF und insgesamt bis zu 100.000 EUR an Plant-for-the-Planet. Unser Ziel: 60.000 Bäume für eine nachhaltige Wiederaufforstung zu pflanzen. Auf der Karte seht ihr, wo das Projekt realisiert wird. Jetzt anschauen und mithelfen."
```

Comment:
@sagararyal do we want to put all the tenant texts into LingoHub and translate them into all of our languages? Because this would do it.

Reply:
If they enable any language then we should have text in that language too.

Reply:
We will get these details from the tenant - confirmed by @sagararyal